apps/explorer/package.json

```diff
@@ -12,11 +12,13 @@
     "preview": "nuxt preview"
   },
   "devDependencies": {
+    "@mapbox/mapbox-gl-geocoder": "^5.0.1",
```

Comment:
Why we need geocoder dependency?

Reply:
https://vue-mapbox-gl.studiometa.dev/guide/installation/

Peer dependency

Reply:
@giovanneguerra For us it's not needed since we are not using geocoding component.